tsconfig.webpack.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.json",
+  "include": [
+    "src"
+  ],
+  "compilerOptions": {
+    "target": "ES2019",
```

Comment:
Isn't the optional chaining introduced with ES2019? Maybe we want to compile to ES6 for React Native

Reply:
Nope, in ES2020. There are resources [here](https://kangax.github.io/compat-table/es2016plus/) and [here](https://github.com/sudheerj/ECMAScript-features).

And I got the idea here: https://stackoverflow.com/a/66422985

More importantly, I tested it with my expo app and it works with `ES2019` 😉